lib/src/models/entity_type.dart

```diff
@@ -0,0 +1,20 @@
+/// Enum class for type of a type of entity
+///
+/// Used for classification of a metric
+class EntityType {
+  /// Entity represent a class, mixin or extension
+  static const EntityType classEntity = EntityType._('class');
+
+  /// Entity represent a class method or constructor, function, getter or setter
+  static const EntityType methodEntity = EntityType._('method');
```

Comment:
Correction: `The entity representing a class method or constructor, function, getter or setter`